bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -95,6 +95,12 @@ cl::opt<bool> ReadPreAggregated(
     "pa", cl::desc("skip perf and read data from a pre-aggregated file format"),
     cl::cat(AggregatorCategory));
 
+cl::opt<std::string>
+    ReadPerfEvents("perf-script-events",
+                   cl::desc("skip perf event collection by supplying a "
+                            "perf-script output in a textual format"),
+                   cl::cat(AggregatorCategory));
```

Comment:
If we don't expect the end user to see this option, let's make it `cl::Hidden` or `cl::ReallyHidden`. Alternatively, we can modify `DataAggregator` interface for testing since you don't invoke the option via CL.